pkg/controller/common/helpers.go

```diff
@@ -292,18 +292,17 @@ func ValidateIgnition(ignconfig interface{}) error {
 		if report := validate2.ValidateWithoutSource(reflect.ValueOf(cfg)); report.IsFatal() {
 			return errors.Errorf("invalid ignition V2 config found: %v", report)
 		}
-		return nil
```

Comment:
any reason to do refactoring in this function? I will suggest when fixing a bug or feature that can be backported to earlier version, it is goo to keep changes limited that is required for the fix. This will help us to avoid introducing accidental (rarely but possible) regression which could have been avoided. Also, nice practice to keep refactoring changes into a separate commit if it doesn't have anything to do with the actual fix/feature a commit is providing.

Reply:
That's from the commit where I had put the validation in `ValidateIgnition` before Jerry said to move it, I can revert that

Reply:
yes, reverting these changes would be good.